main.ahk

```diff
@@ -73,25 +74,50 @@ CB_MISSION(*) {
 }
 
 CB_AUTOKILL(*) {
-    if G.v.step_for_slime {
+    if G.v.seya {
         SendEvent "{w Down}"
+        SendEvent "{q Down}"
         SendEvent "{MButton Down}"
-    }
 
-    SendEvent "{q Down}"
-    loop {
-        loop 50 {
-            EventClick("LButton", 4, , 4)
-        }
-    } until (M.Search(PATH.AUTOKILL) == 0)
-    SendEvent "{q up}"
+        loop {
+            loop 50 {
+                EventClick("g", 4, , )
```

Comment:
g is SBA in default key mapping

custom keymapping will be introduced soon (maybe
thus it would be better to use default keymapping here

Reply:
any suggestion what is SBA  and how to fix it?